.chronus/changes/schemas-unevaluated-props-2025-1-11-14-31-24.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/json-schema"
+  - "@typespec/openapi3"
+---
+
+Updates JsonSchema and Open API 3.1 emitters to use unevaluatedProperties instead of additionalProperties, and updates Open API 3 emitters to match JsonSchema behavior of treating Record<never> as setting "additionalProperties: { not: {} }"
```

Comment:
```suggestion
Updates JsonSchema and Open API 3.1 emitters to use unevaluatedProperties instead of additionalProperties, and updates Open API 3 emitters to match JsonSchema behavior of treating `Record<never>` as setting `additionalProperties: { not: {} }`
```